hack/verify-govet.sh

```diff
@@ -1,83 +1,19 @@
 #!/bin/bash
 
+set -o errexit
 set -o nounset
 set -o pipefail
 
-GO_VERSION=($(go version))
-
-if [[ -z $(echo "${GO_VERSION[2]}" | grep -E 'go1.[4-5]') && -z "${FORCE_VERIFY-}" ]]; then
-  echo "Unknown go version '${GO_VERSION}', skipping go vet."
-  exit 0
-fi
+echo $(go version)
 
 OS_ROOT=$(dirname "${BASH_SOURCE}")/..
-source "${OS_ROOT}/hack/common.sh"
-source "${OS_ROOT}/hack/util.sh"
 
 cd "${OS_ROOT}"
-mkdir -p _output/govet
-
-os::build::setup_env
-
-FAILURE=false
-test_dirs=$(find_files | cut -d '/' -f 1-2 | sort -u)
-for test_dir in $test_dirs
-do
-  go tool vet -shadow=false $test_dir
```

Comment:
I think you know the problem with that pattern :) 
https://github.com/openshift/origin/pull/7593#issuecomment-188413972